interface/interface.c

```diff
@@ -1221,5 +1222,9 @@ void sig_handler(int signo)
 #endif /* TIMER_STATS */
 		rte_exit(EXIT_SUCCESS, "received SIGINT\n");
 	}
+	else if (signo == SIGSEGV){
+		rte_dump_stack();
+		rte_exit(EXIT_FAILURE, "received SIGSEGV\n");
```

Comment:
I think `rte_panic()` calls `rte_dump_stack()` internally for Linux. You can probably combine the lines 1226-1227 into 1 (`rte_panic()`)

Reply:
Updated as per comment. Verified the same.
```
        0         0         0         0         0   ||         0         0         0         0         0
        0         0         0         0         0   ||         0         0         0         0         0
PANIC in sig_handler():
received SIGSEGV
13: [/lib/x86_64-linux-gnu/libc.so.6(clone+0x3f) [0x7fd1d411988f]]
12: [/lib/x86_64-linux-gnu/libpthread.so.0(+0x76db) [0x7fd1d43f06db]]
11: [ngic_dataplane(eal_thread_loop+0x1f1) [0x558a481259e1]]
10: [ngic_dataplane(+0x81b81) [0x558a480a2b81]]
9: [ngic_dataplane(+0x81bad) [0x558a480a2bad]]
8: [ngic_dataplane(iface_process_ipc_msgs+0xdf) [0x558a480aa18f]]
7: [ngic_dataplane(+0x781a4) [0x558a480991a4]]
6: [ngic_dataplane(meter_profile_entry_add+0xb9) [0x558a480a9e19]]
5: [ngic_dataplane(dp_meter_profile_entry_add+0x2a) [0x558a480997fa]]
4: [/lib/x86_64-linux-gnu/libc.so.6(+0x3ef20) [0x7fd1d4036f20]]
3: [ngic_dataplane(+0x8856f) [0x558a480a956f]]
2: [ngic_dataplane(__rte_panic+0xc5) [0x558a48084251]]
1: [ngic_dataplane(rte_dump_stack+0x2e) [0x558a48127c9e]]
/opt/dp/scripts/launch.sh: line 10:    26 Aborted                 (core dumped) ngic_dataplane $EAL_ARGS -- $APP_ARGS
```